src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Isn't this still needed?

Reply:
Same story here: https://github.com/daos-stack/daos/blob/ecca8a53c23cec4f59da930701c91dede2b804b0/ftest.sh#L132